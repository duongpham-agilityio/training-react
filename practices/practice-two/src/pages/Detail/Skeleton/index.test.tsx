import { render } from '@testing-library/react';

// Component
import DetailSkeleton from 'pages/Detail/Skeleton';

describe('DetailSkeleton', () => {
  it('match snapshot', () => {
    const { asFragment } = render(<DetailSkeleton />);

    expect(asFragment()).toMatchSnapshot();
  });
});
