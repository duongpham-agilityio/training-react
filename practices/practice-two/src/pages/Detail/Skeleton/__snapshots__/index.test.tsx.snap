// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailSkeleton match snapshot 1`] = `
<DocumentFragment>
  <section
    class="undefined undefined"
  >
    <div>
      <h2
        class="undefined undefined undefined undefined"
      />
      <div>
        <div>
          <button
            class="undefined undefined
  undefined undefined
  undefined undefined undefined"
            type="button"
          />
          <button
            class="undefined undefined
  undefined undefined
  undefined undefined undefined"
            type="button"
          />
          <button
            class="undefined undefined
  undefined undefined
  undefined undefined undefined"
            type="button"
          />
        </div>
      </div>
    </div>
    <div>
      <div>
        <p
          class="undefined undefined"
        />
        <ul>
          <li
            class="undefined undefined"
          />
          <li
            class="undefined undefined"
          />
          <li
            class="undefined undefined"
          />
        </ul>
      </div>
      <div>
        <div
          class="undefined undefined"
        />
      </div>
    </div>
  </section>
</DocumentFragment>
`;
